### PR TITLE
Feature/push settings screen on start

### PR DESCRIPTION
**Pull request by lynx56**
- Improve not letting to choose less than one wallet (#869426ytv)
- Show welcome screen after update (#86940673f)
- Add are you sure dialog on changed notification settings (#8694270bp)

combine_images-42

### CODE REVIEW

**Comment by stepanLav** [comment]
Task linked: [CU-8694270bp iOS. Add are you sure dialog on changed notification settings](https://app.clickup.com/t/8694270bp)

**Comment by stepanLav** [comment]
Task linked: [CU-869426ytv iOS. Improve not letting to choose less than one wallet](https://app.clickup.com/t/869426ytv)

**Comment by stepanLav** [comment]
Task linked: [CU-86940673f iOS. Show welcome screen after update](https://app.clickup.com/t/86940673f)

File: novawallet/Modules/Notifications/Management/NotificationsManagementInteractor.swift [hunk 1]
```diff
@@ -75,9 +85,44 @@ final class NotificationsManagementInteractor: AnyProviderAutoCleaning {
...
                     self.localSettings = .defined(defaultSettings)
-                    self.presenter?.didReceive(settings: defaultSettings)
+                    self.provideSettings()
+                }
+            }
+        }
+    }
+
+    func fetchWallets() {
+        let fetchWalletsOperation = walletsRepository.fetchAllOperation(with: .init())
```
Comment (ERussel) [inline novawallet/Modules/Notifications/Management/NotificationsManagementInteractor.swift:95]: As we have now a logic (in a separate PR) that syncs locally available wallets with remote do we still need to have this here? I don't think so

File: novawallet/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -367,7 +367,7 @@
 "common.till.date" = "до %@";
 "crowdloan.reward" = "Примерное вознаграждение";
 "staking.bonded.inactive" = "Вы не номинируете и не валидируете";
-"crowdloan.terms.format" = "Я прочёл и соглашаюсь с %@";
+"crowdloan.terms.format" = "Я прочёл и соглашаю��ь с %@";
```
Comment (ERussel) [inline novawallet/ru.lproj/Localizable.strings:370]: Looks like encoding problems

Comment (lynx56) [reply novawallet/ru.lproj/Localizable.strings:370]: all strings were fixed in other PR

File: novawallet/Modules/Notifications/Management/NotificationsManagementInteractor.swift (hunk 1 above)
Comment (lynx56) [reply novawallet/Modules/Notifications/Management/NotificationsManagementInteractor.swift:95]: You are right, now it is not actual

**Review by ERussel — APPROVED** [review]

**Pull request merged**